src/test/java/com/ensao/gi5/lint/rules/violations/EnumUpperRuleTest.java

```diff
@@ -0,0 +1,22 @@
+package com.ensao.gi5.lint.rules.violations;
+
+import com.ensao.gi5.lint.rules.EnumUpperRule;
+import com.ensao.gi5.lint.runner.LinterRunner;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class EnumUpperRuleTest extends LinterRunner {
+
+
+    public static final String FILE_NAME_JAVA = "D:\\Gi5\\Java\\Projet\\ensao-lint\\testFiles\\nameRules\\NameRuleTest.java";
```

Comment:
Test should be independent, this file does exist in only one machine, try to provide test files in the same package